src/apps/Permissions/ManageRolePanel.js

```diff
@@ -311,9 +315,10 @@ class ManageRolePanel extends React.PureComponent {
             <EntitySelector
```

Comment:
Is there a way to select `Any account` here?

Reply:
There's a prop in `EntitySelector` to control this; `ANY_ENTITY` was specifically decided to not be shown here because you can't set a manager to be `ANY_ENTITY`.

Reply:
Ok, so, if I remove the `createVotes` permission from `Any account` I will not be able to set it back right?

Reply:
There's two parts to permissions:

- Permission holders / grantees
- Permission managers

Permission managers have the power to grant new and revoke existing permissions. `ANY_ENTITY` is a concept that only applies to permission holders, and `BURN_ENTITY` is a concept that only applies to permission managers.

Setting the manager to be `ANY_ENTITY` (0xff...ff) is equivalent to saying that address is the manager, and because (hopefully) none of us have the private key to that address, it's lost forever.

A related issue on better documentation for this: https://github.com/aragon/aragonOS/issues/451. We use `BURN_ENTITY` primarily as a standard to help frontends display this information correctly when people want to create discarded / frozen permissions.

-------

Technically we should allow `BURN_ENTITY` in this dropdown, but we'll likely want to design an entire panel around throwing away / freezing a permission forever as it can be pretty serious.